stock_release_channel_partner_delivery_window/models/stock_release_channel.py

```diff
@@ -23,6 +33,13 @@ def filter_release_channel_partner_window(self, partner):
                     hour=channel.process_end_date.hour,
                     minute=channel.process_end_date.minute,
                 )
-            if not partner.is_in_delivery_window(shipment_datetime):
+            if (
+                channel.respect_partner_delivery_time_windows
+                and not any(
+                    picking.sale_id and picking.sale_id.commitment_date
+                    for picking in channel.picking_ids
```

Comment:
@lethuthao2403 when confirming the stock.move, it tries to assign the channel to the picking. It means that at this phase of code, the picking has no relation to the channel yet. so, you can not rely on `channel.picking_ids`.